tests/MyTests.cpp

```diff
@@ -38,21 +38,40 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <limits>
 #include <type_traits>
 
-#include "Exception.hpp"
-#include "MyLibrary.hpp"
+#include "../src/Exception.hpp"
+//#include "../src/MyLibrary.hpp"
+#include <iostream>
+#include <dcmtk/dcmpstat/dcmpstat.h>
+#include "../src/metadataEditor.cpp"
+#include "../src/metadataEditor.hpp"
 
 using namespace cpp_template;
 
 // This tests the output of the `get_nth_prime` function
 TEST_CASE("correct primes are returned", "[primes]") {
-  CHECK(get_nth_prime(0) == 2);
-  CHECK(get_nth_prime(1) == 3);
-  CHECK(get_nth_prime(2) == 5);
-  CHECK(get_nth_prime(854) == 6619);
+
+  DcmFileFormat image;
+  OFCondition status = image.loadFile("../DICOM_Images/1-1copy.dcm");
```

Comment:
I can see that you've copied the example to check that Dcm is working, but in the longer term it would be better to generate this file, rather than storing in the repo. Git isn't very useful for  binary files. But this can wait for now